report_alternative_layout/models/report_paperformat.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ReportPaperformat(models.Model):
+    _inherit = "report.paperformat"
+
+    apply_alternative_layout = fields.Boolean(
+        help="If selected, the alternative layout will be applied in the printed "
+        "report.",
+    )
+    show_address_in_header = fields.Boolean(
+        compute="_compute_show_address_in_header",
+        help="If selected, the report header will be shown on every page of the "
+        "report output.",
+        readonly=False,
+        store=True,
+    )
+
+    @api.depends("apply_alternative_layout")
+    def _compute_show_address_in_header(self):
+        for rec in self:
+            if rec.apply_alternative_layout:
+                continue
+            rec.show_address_in_header = False
```

Comment:
I don't understand this logic. Why continue if the report is apply_alternative_layout? No need to make True?

Reply:
I want this logic to ensure that when apply_alternative_layout is set to false, show_address_in_header is also set to false.

In other words, I want to prevent the case
- apply_alternative_layout = false 
- show_address_in_header = true.

Reply:
If that is the case, I think this is enough.
```
if not rec.apply_alternative_layout:
   rec.show_address_in_header = False
```